### PR TITLE
ReversingLabs - Add proxy support

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.yml [hunk 1]
```diff
@@ -25,6 +25,30 @@ configuration:
...
+  name: http_proxy
+  type: 0
+  required: false
+- display: HTTP proxy username (if needed)
+  name: http_proxy_username
+  type: 0
+  required: false
+- display: HTTP proxy password (if needed)
+  name: http_proxy_password
+  type: 4
```
Comment (RosenbergYehuda) [inline Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.yml:38]: Here too, we recommend using type 9.

Note that you already have a use of type 9 above, in order to distinguish between the two, call the second one a different name.
For example: name: credentinals2, and then in the python you can enter its values like this: 

demisto.params().get("credentials2", {}).get("identifier")

**Review by RosenbergYehuda — COMMENTED** [review]
Great job @MislavReversingLabs!
I added some comments, please take a look and let me know if anything isn't clear enough.

File: Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.yml [hunk 2]
```diff
@@ -40,6 +40,30 @@ configuration:
...
+  name: http_proxy
+  type: 0
+  required: false
+- display: HTTP proxy username (if needed)
+  name: http_proxy_username
+  type: 0
+  required: false
+- display: HTTP proxy password (if needed)
+  name: http_proxy_password
+  type: 4
```
[the remark this answers is not in the document]
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.yml:53]: Type 4 is also masked as a password. What is the difference between 4 and 9?

[the remark this answers is not in the document]
Comment (RosenbergYehuda) [reply Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.yml:53]: That's correct. Both, type 4 and type 9 are encrypted.
The uniqueness of type 9 is that it allows the user to switch to credentials, as you can see in ReversingLabs TitaniumCloud.

**Comment by RosenbergYehuda** [comment]
> @RosenbergYehuda What does this error mean? `13:02:25.208953 ERROR: Label Partner-Approved was not added to PR: 28993. Please ask the partner to review the code and add the label after his approval.`

Its something new to protect content of partners, like yours.
You can ask @edik24 for more details, he is the one that added it.
anyway, I added the new label "partner approved" and the error will disappear.

**Review by RosenbergYehuda — APPROVED** [review]

**Comment by MislavReversingLabs** [comment]
> > @RosenbergYehuda What does this error mean? `13:02:25.208953 ERROR: Label Partner-Approved was not added to PR: 28993. Please ask the partner to review the code and add the label after his approval.`
> 
> Its something new to protect content of partners, like yours. You can ask @edik24 for more details, he is the one that added it. anyway, I added the new label "partner approved" and the error will disappear.

Great, thank you!

**Pull request merged**